### PR TITLE
Updating README for the correct usage operations

### PR DESCRIPTION
**Pull request by vishrutshah**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vishrutshah__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: azure_mgmt_storage/README.md [hunk 1]
```diff
@@ -48,17 +48,14 @@ client = Azure::ARM::Storage::StorageManagementClient.new(credentials)
...
 params = Models::StorageAccountCreateParameters.new
-params.properties = props
 params.location = 'westus'
 sku = Models::Sku.new
 sku.name = 'Standard_LRS'
 params.sku = sku
 params.kind = Models::Kind::Storage
 
-promise = client.storage_accounts.create('some_existing_resource_group', 'newstorageaccount', params)
+promise = client.storage_accounts.create_async('some_existing_resource_group', 'newstorageaccount', params)
```
Comment (veronicagg) [inline azure_mgmt_storage/README.md]: should we be including in this sample code, just "create" that returns the storage account directly now, and not a promise anymore (unless calling _async)?

**Comment by veronicagg** [comment]
@vishrutshah thanks! Made a comment inline. Let's make sure to execute the sample code before publishing. Also, it'd be helpful to mention, if not there already, to look into some of the tests for usage, for the ones that exist; and search for "ruby" on the Azure samples site for samples as well.
We also have more gems now, so we may need to update the list in the readme with the new gems.

**Comment by vishrutshah** [comment]
@veronicagg As per our offline talk, I've updated the README to include all released gems and in the "Getting Started Samples" added link to Ruby samples from Azure-Samples org. Please have a look when you get a chance. Thanks!

**Pull request merged**